script/tool/README.md

```diff
@@ -118,6 +118,32 @@ cd <repository root>
 dart run ./script/tool/bin/flutter_plugin_tools.dart update-excerpts --packages plugin_name
 ```
 
+### Update CHANGELOG and Version
+
+`update-release-info` will automatically update the version and `CHANGELOG.md`
+following standard repository style and practice. It can be used for
+single-package updates to handle the details of getting the `CHANGELOG.md`
+format correct, but is especially useful for bulk updates across multiple packages.
+
+For instance, if you add a new analysis option that requires production
+code changes across many packages:
+
+```sh
+cd <repository root>
+dart run ./script/tool/bin/flutter_plugin_tools.dart update-release-info \
+  --version=minimal \
+  --changelog="Fixes violations of new analysis option some_new_option."
+  --run-on-changed-packages
```

Comment:
Can `--run-on-changed-packages` be the default behavior for this tool, so people don't have to "remember" to pass it?

Reply:
It would require some re-plumbing to allow a command to change the default package selection behavior, and it would make it inconsistent with all the other commands. But then, it's a different kind of command than most of the others.

I'll play around with that tomorrow.

Reply:
I came up with a different solution when I woke up this morning :) Instead of making this command have non-standard package targeting, I just made `minimal` skip any package that has no changes at all. That makes it even easier to use, since you don't have to worry about the base treeish that `run-on-changed-packages` is using.

You'll still need to explicitly target for the other modes, but I don't expect the other modes to actually be useful in a bulk-update setting.